src/mixins/filing-template-mixin.ts

```diff
@@ -491,8 +476,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
     // store Entity Snapshot
     this.setEntitySnapshot(entitySnapshot)
 
-    // *** FUTURE: remove this fallback when Filings UI provides this value
-    if (!filing.correction.type) filing.correction.type = CorrectionErrorTypes.STAFF
+    // safety check (should never happen)
+    if (!filing.correction.type) filing.correction.type = CorrectionErrorTypes.CLIENT
```

Comment:
This shows a more complete filing than STAFF. If this is wrong then can delete and recreate this correction.